solo-chains/runtime/starlight/Cargo.toml

```diff
@@ -84,7 +84,6 @@ pallet-scheduler = { workspace = true }
 pallet-session = { workspace = true }
 pallet-society = { workspace = true }
 pallet-staking = { workspace = true }
-pallet-state-trie-migration = { workspace = true }
```

Comment:
I feel like a few more should dissappear from this right?

Reply:
The identity migration is inside `runtime-common`, so no extra dependency for that one